pkg/deploy/testdata/sadPathDeploymentFileTrafficManagement.yaml

```diff
@@ -0,0 +1,109 @@
+version: v1
+kind: kubernetes
+application: app
+# Map of Deployment Targets, this is set up in a way where
+# we can do multi-target deployments (multi-region or multi-cluster)
+targets:
+  # This in the name of a deployment, underneath it are its configuration
+  test1:
+    account: account1
+    namespace: dev
+    strategy: strategy1
+  test2:
+    account: account2
+    namespace: qa
+    strategy: strategy2
+    constraints:
+      dependsOn: ["test1"]
+      beforeDeployment:
+        - pause:
+            untilApproved: true
+trafficManagement:
+  - targets: ["test1"]
+    smi:
+      rootServiceName: "qa-service-root"
```

Comment:
`rootServiceName` is the only one that's required - would just swap in `canaryServiceName` here.